climpred/tests/test_graphics.py

```diff
@@ -9,6 +9,7 @@ def test_mpi_pm_plot_bootstrapped_skill_over_leadyear():
     """
     da = load_dataset('MPI-PM-DP-1D').isel(area=1, period=-1)
     PM_da_ds1d = da['tos']
+    PM_da_ds1d['lead'].attrs['units'] = 'years'
```

Comment:
Do we now need to set this every time. Cannot we infer this when init is int? Because if we need to set this now, this will be a breaking change I’d prefer to avoid.

Reply:
Now you need to set this everywhere... see below

Reply:
See discussion in main PR.